openapi/sinch.sms/Module.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+This is a generated connector for [Sinch SMS API v2.0](https://www.sinch.com/) OpenAPI specification. 
+
+Sinch SMS API is the one of the easiest APIs we offer and enables you to add fast and reliable global SMS to your 
+applications. It allows for sending single messages, scheduled batch sending, message templates and more.
```

Comment:
We can merge this line to previous line.

Reply:
Merged. thanks